docs/manual/creating-a-checker.tex

```diff
@@ -1783,6 +1783,9 @@
 already reported the bug, and you want to continue using the checker on a
 large codebase without being inundated in stack traces.
 
+\item \code{-AdumpOnErrors}: Outputs the stack trace of the program when the
```

Comment:
I think you can remove the `of the program` and shorten to `output a stack trace when reporting errors or warnings.`